zipline/algorithm.py

```diff
@@ -789,6 +825,47 @@ def fetch_csv(self, url,
                   symbol_column=None,
                   special_params_checker=None,
                   **kwargs):
+        """Fetch a csv from a remote url.
```

Comment:
Could we mention that it automatically registers the returned source with the data portal?